Index.html

```diff
@@ -1,12 +1,126 @@
+    <link rel="stylesheet" href="./Styles/index.css" />
+    <title>moustrack | new roads awaits you</title>
+    <link rel="shortcut icon" type="images/gif" href="./Assets/world.webp" />
+  </head>
+  <body>
+    <header class="header">
+      <div><img src="./Assets/logo-black.webp" alt="Mouse-Track" /></div>
+
+      <div class="item">
+        <ul class="nav-items">
+          <li>HOME</li>
+          <li>NEW ROADS</li>
+          <li>TRACKS</li>
+          <li>ABOUT</li>
+        </ul>
+
+        <ul class="portfolio">
+          <li>VOLVER AL PORTFOLIO</li>
+        </ul>
+      </div>
+    </header>
+
+    <hr class="border">
+
+    <div> 
+    <ul class="list-2">
+        <ul>
+          <li class="title-2">YOUR JOURNEY BEGINS HERE</li>
+          <li class="button1">START NOW</li>
+        </ul>
+        <ul>
+         <li class="logo-size"><img src="./Assets/world.webp" alt="Mouse-Track" /></li>
+         <li class="title-2b">millions of competitors around the world are already warming-up for the biggest event of their lives.</li>
+        </ul>  
+    </ul>
+    </div> 
+
+  
+  <hr class="border">
+
+  <div class="list-3">
+    <ul>
+
+      <li class="font-3A">NEW ROADS AWAIT YOU</li>
+      
+      <li class="font-3B">From mountains to cities, moustrack tours are full of wonderful natural and artificial settings.</li>
     
-</body>
-</html>
\ No newline at end of file
+      <li class="button2">LEARN MORE</li>
+    
+    </ul>
+  </div>
+
+  <hr class="border"></hr>
+   
+  <div>
+    <ul class="align">
+      <li class="list-4 button3">CHOOSE YOUR DESTINATION</li>
+    </ul>
+
+    <ul>
+      <li class="move-left">NORTH AMERICA</li>
+      <li class="button1">MORE INFO</li>
+    </ul>
+
+      <hr class="border"></hr>
+
+    
+      
+      <ul class="move-right">
+        <li>SOUTH AMERICA</li>
+         <li class="button1">MORE INFO</li>
+      </ul>
+  
+
+      <hr class="border"></hr>
+
+    <ul>
+      <li class="move-left">EUROPA</li>
+      <ul class="button1">
+      <li>MORE INFO</li>
+      </ul>
+    </ul>
+
+  </div>
+  
+  <hr class="border"></hr>
+
+    <div>
+
+      <div class="list-5">
+      <ul>
+
+       <ul class="button1">
+      <li>
+        <img src="./Assets/logo-black.webp" alt="Mouse-Track" />
+      </li>
+      <li>YOUR GO-TO TRAVEL MANAGER</li>
+      </ul>  
+
+        
+       
+        <li>
+        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eveniet cum
+        perspiciatis accusantium ratione molestias corrupti. Quo reiciendis
+        quod, totam illum harum omnis culpa expedita, tempora minima deserunt
+        earum eum molestiae, voluptates laboriosam consequatur excepturi
+        provident?
+        </li>
+        
+
+      <li class="move-right">designed by Hy-An Multimedia Group</li>
+
+      <li class="move-right">images by unsplash.com</li>
+      </ul>
+      </div>
+
+      
+    </div>
+  </body>
+</html>
```

Comment:
1) Don't use numbers in your class names, in here and in the SCSS file. Use more descriptive names.
2) Don't abuse of the vertical space, makes the code unreadable and hard to connect / understand for other people